loginCall.js

```diff
@@ -22,8 +26,8 @@ const loginCall = (loginData, cb) => {
       function page(records, fetchNextPage) {
         // This function (`page`) will get called for each page of records.
         records.forEach(function(record) {
-          result.push(record.get("Name"));
-          result.push(record.get("ID"));
+        const { Name, ID}  = record.fields;
+          return cb(null, { Name , ID});
```

Comment:
definitely `camelCase` javascript variables

Reply:
this is a bit dangerous as a pattern, it works in this case because we only have one loop of `records.forEach`. 

i think it would be better if it was:
```
	const user = records[0]
	return cb(null, {
		name: user.Name,
		id: user.ID
	})
})
```

Reply:
Great shout

Reply:
re: `camelCase` - it appears we can't change `{ Name, ID}` to `{ name, id}` for destructuring purposes as that's how the keys are formatted  in the records[0].fields object?

Reply:
yah you dont need to destructure:
		name: user.Name,
		id: user.ID

will do it

if you wanted to desctructure you could do:
```
const { Name: name, ID: id } = record[0]
```